src/wallet/wallet.h

```diff
@@ -1056,9 +1056,10 @@ class CWallet : public CCryptoKeyStore, public CValidationInterface
     CPubKey GenerateNewHDMasterKey();
     
     /* Set the current HD master key (will reset the chain child index counters)
-       If possibleOldChain is provided, the parameters from the old chain (version)
-       will be preserved. */
-    bool SetHDMasterKey(const CPubKey& key, CHDChain *possibleOldChain = nullptr);
+       Sets the master keys version based on the current wallet version (so the
```

Comment:
nit: you missed one of the grammar errors: s/keys/key's/ . There's only one of them.

(sorry - normally I wouldn't nit for grammar, but this could be misleading - hence my earlier suggestion to reword)